src/lib/components/seo.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+	import { page } from '$app/state'
+
+	interface Props {
+		title: string
+		description: string
+	}
+
+	const { title, description }: Props = $props()
+	const url = page.url.href
+</script>
+
+<svelte:head>
+	<title>{title}</title>
+	<meta name="description" content={description} />
+	<link rel="canonical" href={url} />
+	<meta name="keywords" content="jogjajs, javascript, jogja, komunitas javascript di yogyakarta" />
+	<meta name="robots" content="index,follow" />
+	<meta name="googlebot" content="index,follow" />
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta name="twitter:site" content="@jogjajs" />
+	<meta name="twitter:title" content={title} />
+	<meta name="twitter:description" content={description} />
+	<meta name="og:title" content={title} />
+	<meta name="og:url" content={url} />
+	<meta name="og:type" content="website" />
+	<meta name="og:description" content={description} />
+	<meta name="og:image" content="/og.png" />
+	<meta name="og:site_name" content="Official JogjaJS website" />
```

Comment:
```suggestion
	<meta property="og:title" content={title} />
	<meta property="og:url" content={url} />
	<meta property="og:type" content="website" />
	<meta property="og:description" content={description} />
	<meta property="og:image" content="/og.png" />
	<meta property="og:site_name" content="Official JogjaJS website" />
```

The Open Graph tags (og:*) should use `property`, not `name`.

Ref: https://ogp.me/#metadata